src/library/scala/Option.scala

```diff
@@ -63,6 +63,27 @@ object Option {
  *  This allows for sophisticated chaining of $option values without
  *  having to check for the existence of a value.
  *
+ * These are useful helper functions that exist for both $some and $none.
```

Comment:
nitpick: I would suggest replacing "helper functions" with "methods". these aren't "helper"s IMO, I consider a helper to something, usually private or not top level, that you don't call directly, that exists only to "help" the thing that you do call directly. and it's pretty much always better to say "method" instead of "function" unless you mean a function value.,

Reply:
Yes, I agree.  That was a imprecise use of the phrase "helper functions".

Reply:
The helper's function is to help 'er function.